example/src/PaymentMethodsView.js

```diff
@@ -27,7 +27,7 @@ const PaymentMethods = () => {
   const platformSpecificPayment =
     Platform.OS === 'ios' ? 'Apple Pay' : 'Google Pay';
   const platformSpecificType =
-    Platform.OS === 'ios' ? 'applepay' : 'paywithgoogle';
+    Platform.OS === 'ios' ? 'applepay' : 'googlepay';
```

Comment:
Other places I see we kept `paywithgoogle` but not here. I don't know what this line does so not 100% sure how removing `paywithgoogle` will affect here.

Reply:
fixed [9b8737b](https://github.com/Adyen/adyen-react-native/pull/105/commits/9b8737ba6198192e2c97dcdcba07765a7954eab7)